test/passes/br_if_to_try.cpp

```diff
@@ -0,0 +1,11 @@
+extern bool getBoolean();
```

Comment:
I assume this is just to document where `br_if_to_try.wasm` comes from? If so, it would be good to add a comment with the command to reproduce `br_if_to_try.wasm` in the future.

Reply:
Oh, even better to go with @aheejin's suggested test case.